src/main/java/io/redisearch/Schema.java

```diff
@@ -105,11 +105,19 @@ public void serializeRedisArgs(List<String> args) {
         private final String separator;
 
         public TagField(String name) {
-        	this(name, DEFAULT_SEPARATOR);
+        	this(name, DEFAULT_SEPARATOR, false);
```

Comment:
why did you need this change?

Reply:
For sure it's not strictly required. I felt like chaining the constructors was a bit ugly and going directly to the maximum parameterized constructor was best. I'm not strongly attached to this change tho. Just my personal preference.

Reply:
I prefer it the other way so if we change the default it will effect both.

Reply:
sure... makes sense